src/Sushi.Mediakiwi/Framework/Api/MediakiwiJavascriptEvent.cs

```diff
@@ -4,8 +4,8 @@ public enum MediakiwiJSEvent
     {
         none = 0,
         change = 1,
-        click = 1,        
-        blur = 2,
-        keyup = 3,
+        click = 2,        
+        blur = 3,
+        keyup = 4,
     }
```

Comment:
Is there a reason these enum values do not start with an uppercase character?

Reply:
Not that i'm aware of, this was existing code albeit with a double value (1) which i fixed.
i'll check that the JSON output remains the same and will then rename them.